### PR TITLE
Fix misc typos

### PR DESCRIPTION
**Pull request by bretello**

### CODE REVIEW

File: server/sshd_test.go [hunk 1]
```diff
@@ -80,6 +80,6 @@ func Test_sshd_DisallowSession(t *testing.T) {
 
 	_, err = client.NewSession()
 	if err == nil || !strings.Contains(err.Error(), "unsupported channel type") {
-		t.Fatalf("expect unsupported channle type error but got %v", err)
+		t.Fatalf("expect unsupported channele type error but got %v", err)
```
Comment (owenthereal) [inline server/sshd_test.go]:
```suggestion
		t.Fatalf("expect unsupported channel type error but got %v", err)
```

File: server/sshproxy_test.go [hunk 2]
```diff
@@ -125,7 +125,7 @@ func Test_sshProxy_dialUpstream(t *testing.T) {
 			}
 			_, err = client.NewSession()
 			if err == nil || !strings.Contains(err.Error(), "unsupported channel type") {
-				t.Fatalf("expect unsupported channle type error but got %v", err)
+				t.Fatalf("expect unsupported channele type error but got %v", err)
```
Comment (owenthereal) [inline server/sshproxy_test.go]:
```suggestion
				t.Fatalf("expect unsupported channel type error but got %v", err)
```

**Review by owenthereal — COMMENTED** [review]
Thanks. Fixing two typos

**Comment by bretello** [comment]
Thanks @owenthereal, included those two changes in the last force-push.

**Pull request merged**